### ISSUE TITLE
SRL model trained with ELMo Small

### ISSUE THREAD

**Issue by tbmihailov** [allenai/allennlp#2356]
Is the SRL model trained with ELMo Small [1] available for download for use with AllenNLP? If not, do you have an estimation of how much time (epochs and time) would it take to train it from scratch with ELMo Small? 

[1] https://allennlp.org/elmo

**Comment by schmmd** [comment]
@tbmihailov I don't think we have such a model available and I think it takes several days to train, but @DeNeutoy can follow up with a more authoritative answer.

**Comment by schmmd** [comment]
@DeNeutoy will follow up.
